src/inverseRobot.js

```diff
@@ -7,7 +7,25 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+  const propertyValues = Object.values(robot);
+  let current;
```

Comment:
The variables 'propertyValues' and 'current' are not necessary and can be removed. You can access the values directly in the loop.